src/Executor/ManifestExecutor.php

```diff
@@ -110,6 +102,44 @@ public function runModuleManifestExecution(
         return $this->integratorLockWriter->storeLock($lockedModules);
     }
 
+    /**
+     * @param array<mixed> $unappliedManifestByType
+     * @param string $manifestType
+     * @param string $moduleName
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     * @param bool $isDry
+     *
+     * @return void
+     */
+    protected function applyManifestsByType(
+        array $unappliedManifestByType,
+        string $manifestType,
+        string $moduleName,
+        InputOutputInterface $inputOutput,
+        bool $isDry
+    ): void {
+        try {
+            $manifestExecutor = $this->resolveExecutor($manifestType);
+        } catch (RuntimeException $runtimeException) {
+            return;
+        }
+
+        foreach ($unappliedManifestByType as $manifestHash => $unappliedManifest) {
+            try {
+                if ($manifestExecutor->apply($unappliedManifest, $moduleName, $inputOutput, $isDry)) {
+                    $lockedModules[$moduleName][$manifestType][$manifestHash] = $unappliedManifest;
+                }
+            } catch (Exception $exception) {
```

Comment:
if you throw ManifestApplyingException you need to catch it. don't catch all Exceptions

Reply:
Yes, you are right 
I`m done here with the linked issue https://spryker.atlassian.net/browse/SDK-1431